pkg/gardenlet/apis/config/validation/validation.go

```diff
@@ -72,3 +81,37 @@ func validateSNI(sniPath *field.Path, sni *config.SNI) field.ErrorList {
 
 	return allErrs
 }
+
+// ValidateShootControllerConfiguration validates the shoot controller configuration.
+func ValidateShootControllerConfiguration(cfg *config.ShootControllerConfiguration, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	if cfg.ConcurrentSyncs != nil {
+		allErrs = append(allErrs, apivalidation.ValidateNonnegativeField(int64(*cfg.ConcurrentSyncs), fldPath.Child("concurrentSyncs"))...)
+	}
+
+	if cfg.ProgressReportPeriod != nil {
+		allErrs = append(allErrs, apivalidation.ValidateNonnegativeField(int64(cfg.ProgressReportPeriod.Duration), fldPath.Child("progressReporterPeriod"))...)
+	}
+
+	if cfg.RetryDuration != nil {
+		allErrs = append(allErrs, apivalidation.ValidateNonnegativeField(int64(cfg.RetryDuration.Duration), fldPath.Child("retryDuration"))...)
+	}
+
+	if cfg.SyncPeriod != nil {
+		allErrs = append(allErrs, apivalidation.ValidateNonnegativeField(int64(cfg.SyncPeriod.Duration), fldPath.Child("syncPeriod"))...)
+	}
+
+	if cfg.DNSEntryTTLSeconds != nil {
+		const (
+			dnsEntryTTLSecondsMin = 30
+			dnsEntryTTLSecondsMax = 3600
```

Comment:
Where is this value coming from?

Reply:
From space ;-) Jokes aside, there is no specific reason why I chose it. Probably it's even too large. Maybe we should set the maximum to something like `300` or `5m`. Otherwise, the control plane migration won't work smoothly, I assume. I've set to to `300` now. 
/cc @plkokanov @swilen-iwanow 

Hope that is still fine for your use-cases @schrodit @MartinWeindel

Reply:
300 is fine for the OTC use case.
@vasu1124 do you know any other dns service that has a higher min ttl?

Reply:
Let's go ahead with `300` then. we can increase it later if needed.

Reply:
Sorry for my late answer, but I just remember Jerry Jia told me, that in the case of Alicloud at least 600 seconds are needed to avoid buying enterprise service for DNS.

Reply:
OK, thanks. I'll change it to `600` then.

Reply:
Wait a sec, actually it is already on `600`. please check.